File: Data Structures & Algorithms/majority-element/submission-0.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        // Naive approach is to use hashing. 
        unordered_map<int, int> mpp; 
        for(int i = 0; i < nums.size(); i++)mpp[nums[i]]++; 
        for(const auto &pair: mpp){
            if (pair.second > nums.size()/2)return pair.first; 
        }
        return -1; // Place holder. 
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j, n = 0, len(nums) 
        for i in range(n): 
            if (nums[i] != nums[j]): 
                j += 1
                nums[j] = nums[i] 
        return j + 1
